site/docs/resources/about.md

```diff
@@ -0,0 +1,53 @@
+# About grammY
+
+## What is grammY?
+
+grammY is a piece of software that you can use when you want to program your own chat bot for [the Telegram messenger](https://telegram.org).
+When you make bots, you will notice that some parts of this process are tedious and always the same.
+grammY does the heavy lifting for you and makes it super simple to create a bot.
+
+## When was grammY created?
+
+The first publish of grammY code was in late March, 2021.
+It reached the first stable version a few weeks later.
+
+## Who develops grammY?
+
+grammY is developed by a team of volunteers who work on the core library, the docs, and the remaining ecosystem in their free time.
+You can find a list of all contributors [in the README of our repository](https://github.com/grammyjs/grammY#contributors-).
+
+Feel free to take contact [in the group chat](https://t.me/grammyjs)!
+If you speak Russian, you can also join us [here](https://t.me/grammyjs_ru)!
+
+## How is grammY developed?
+
+grammY is completely free and open-source software.
```

Comment:
I'm more familiar with something being "free software", as in the uncountable stuff called OSS, rather than a product. Don't have strong opinions, though, I'll leave it up to you.